src/main/java/com/jfrog/ide/idea/scan/data/applications/ModuleConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.jfrog.ide.idea.scan.data.applications;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+
+@Getter
+public class ModuleConfig {
```

Comment:
This class does not contain any setters or an all-args constructor. Are you sure that deserialization into it would work?
Same question for ScannerConfig.